From bb934f59cdd79af72d1719a41e1c501959baec5d Mon Sep 17 00:00:00 2001
From: Marius Gripsgard <marius@ubports.com>
Date: Fri, 14 Aug 2020 20:38:25 +0200
Subject: [PATCH 2/2] Fix warning on newer gcc

This warning is not something we can do anything about as glib by
default do this sin all the time.
---
 src/adapters/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/adapters/CMakeLists.txt b/src/adapters/CMakeLists.txt
index a0d3a52..793237b 100644
--- a/src/adapters/CMakeLists.txt
+++ b/src/adapters/CMakeLists.txt
@@ -71,6 +71,8 @@ add_library(
     ${REPOWERD_ADAPTER_SRCS}
 )
 
+target_compile_options(repowerd-adapters PRIVATE -Wno-cast-function-type)
+
 target_link_libraries(
     repowerd-adapters
 
-- 
2.25.1

